core/src/Kokkos_TaskScheduler.hpp

```diff
@@ -418,17 +418,21 @@ class TaskScheduler
   KOKKOS_INLINE_FUNCTION
   TaskScheduler() : m_track(), m_queue(0) {}
 
-  KOKKOS_INLINE_FUNCTION_DEFAULTED
-  TaskScheduler( TaskScheduler && rhs ) = default ;
+  KOKKOS_INLINE_FUNCTION
+  TaskScheduler( TaskScheduler && rhs )
+    : m_track( rhs.m_track ), m_queue( rhs.m_queue ) {}
 
-  KOKKOS_INLINE_FUNCTION_DEFAULTED
-  TaskScheduler( TaskScheduler const & rhs ) = default ;
+  KOKKOS_INLINE_FUNCTION
+  TaskScheduler( TaskScheduler const & rhs )
+    : m_track( rhs.m_track ), m_queue( rhs.m_queue ) {}
 
-  KOKKOS_INLINE_FUNCTION_DEFAULTED
-  TaskScheduler & operator = ( TaskScheduler && rhs ) = default ;
+  KOKKOS_INLINE_FUNCTION
+  TaskScheduler & operator = ( TaskScheduler && rhs )
+    { m_track = rhs.m_track ; m_queue =  rhs.m_queue ; return *this ; }
 
-  KOKKOS_INLINE_FUNCTION_DEFAULTED
-  TaskScheduler & operator = ( TaskScheduler const & rhs ) = default ;
```

Comment:
Why are these not defaulted anymore?

Reply:
Have had problems with CUDA 9 and explicitly defaulted constructors and assignment operators.  These are trivial so just inserted the implementation.

Reply:
Okay, sounds good